biztalk/install-and-config-guides/configure-biztalk-server.md

```diff
@@ -243,6 +243,38 @@ BAM alerts require BAM tools to be enabled.
     |Windows group|You can keep the default group name, or enter something custom. |
     |Windows SharePoint Services Adapter Web site|Select the Web site that hosts the Windows SharePoint Service Adapter web service.|
 
+### Configure the REST APIs
+>[!NOTE]
+>REST APIs configuration is available start from **BizTalk Server 2020**.
+
+
+1. Select **REST APIs**.
+2. Configure the following:
+
+    |Use this|To do this|
+    | --- | --- |
+    |Enable BizTalk REST APIs | Check this if you want enable BizTalk REST APIs. | 
+    |Web service accounts | Specify the account used to run the IIS services. <ul><li>Management Service: REST APIs for BizTalk Server artifacts management. The default URI for **Management Service** APIs is http(s)://localhost/BizTalkManagementService. </li><li>Operational Data Service: Service providing information on the instances and messages flowing through your BizTalk Server environment. The operational data feed is the same data you get looking at Group Hub in BizTalk Server Administration.The default URI for **Operational Data service** APIs is http(s)://localhost/BizTalkOperationalDataService.</li></ul>|
+    |Windows groups | BizTalk REST APIs use Windows authentication, specify the Windows groups allowed to access the APIs here. <ul><li>Management REST APIs User Role: Specify this as an extra layer of security check or keep default, because actually windows user who *request* (not user hosting the service) to management REST APIs will take the access challenge. </li><li>Operational Data REST APIs User Role: Specify user group allowed to access.</li>|
+    |REST APIs website|Select the Web site to host the both the REST APIs. |
+
+### Configure BizTalk TMS
+    BizTalk TMS is a service that refreshes the authentication tokens used by BizTalk, it is a prerequisite for some adapters, i.e. Office 365 series adapters.  
```

Comment:
In the markdown preview in github, the formatting for this statement is looking different from normal text. Can you please check?

Reply:
Fixed.